hw/lint/tools/veriblelint/rules.vbl

```diff
@@ -13,7 +13,8 @@ line-length=length:150
 explicit-parameter-storage-type=exempt_type:string
 
 # localparam can be both ALL_CAPS and CamelCase according to our style
-parameter-name-style=localparam_style:CamelCase|ALL_CAPS
+# lower_case is added for legacy UVM reasons
+parameter-name-style=localparam_style:CamelCase|ALL_CAPS|lower_case
```

Comment:
For parameters, we only have CamelCase and ALL_CAPS.  For macros, we wanted to allow ALL_CAPS and lower_case.

Reply:
Ok, I see. If this is needed for macros only, I'll remove this change again.

Unfortunately, the macro name style cannot be configured yet in Verible. I've opened https://github.com/google/verible/issues/378 for that.